proto/skaffold.proto

```diff
@@ -640,6 +681,10 @@ enum StatusCode {
     DEPLOY_PARSE_MANIFEST_IMAGES_ERR = 1021;
     // Helm config `createNamespace` not available
     DEPLOY_HELM_CREATE_NS_NOT_AVAILABLE = 1022;
+
+    // Test Errors
+
+    TEST_UNKNOWN_ERROR = 1101;
```

Comment:
Is this same as `TEST_UNKNOWN`?  I don't think we need 2 unknown error codes.

Reply:
This is just a place holder or a catch all until we finalize the error codes. I am open to adding this later.

Reply:
Removed the TEST_UNKNOWN_ERROR error code.